### PR TITLE
Delta chat IPC

### PR DESCRIPTION
**Pull request by cyBerta**
corresponding deltachat pr: https://github.com/deltachat/deltachat-android/pull/2088

### CODE REVIEW

**Comment by r10s** [comment]
i did a test with this pr and the [corresponding AndroidYggmail pr](https://github.com/deltachat/AndroidYggmail/pull/15) in and android9 emulator:
- the app is started as expected (tested with already installed app only)
- however, the email address arrices as domain only, the password is set to `delta` however

**Comment by cyBerta** [comment]
did you double check that yggmail is really started and has generated an address? Seems like a bug on AndroidYggmail.

**Comment by cyBerta** [comment]
Did you receive a toast in YggmailAndroid stating that the email address was copied?

**Comment by cyBerta** [comment]
I wonder if the lib was built correctly, too

**Comment by r10s** [comment]
> Did you receive a toast in YggmailAndroid stating that the email address was copied to the clipboard?

no, that toast did not appear. i also double checked on a real device, same.

> i wonder if the lib was built correctly, too

yip, this is also what i am wondering. build went through without further errors - but the log is completely empty, so there is sth. wrong, indeed.

can you provide an apk for testing - i mean the most important part is currently to get https://github.com/deltachat/deltachat-android/pull/2088 in - we can target the yggmail issues later

**Comment by cyBerta** [comment]
> can you provide an apk for testing - i mean the most important part is currently to get [deltachat/deltachat-android#2088](https://github.com/deltachat/deltachat-android/pull/2088) in - we can target the yggmail issues later

https://share.riseup.net/#BNHkQAhj_3qK2ym3xgGsNQ

the link is only valid for 12 h

File: app/src/androidTest/java/chat/AndroidYggmail/delta/ExampleInstrumentedTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.cyberta;
+package chat.AndroidYggmail.delta;
```
Comment (adbenitez) [inline app/src/androidTest/java/chat/AndroidYggmail/delta/ExampleInstrumentedTest.java:1]: shouldn't the name change here to lower case as well? it doesn't look good with camel case

Comment (cyBerta) [reply app/src/androidTest/java/chat/AndroidYggmail/delta/ExampleInstrumentedTest.java:1]: this package name is completely wrong :D

**Pull request merged**